tactics/rewrite.ml

```diff
@@ -954,10 +954,8 @@ let fold_match ?(force=false) env sigma c =
         else case_nodep)
     in
     match Ind_tables.lookup_scheme sk ci.ci_ind with
-    | Some cst ->
-        dep, pred, cst
-    | None ->
-      raise Not_found
+    | Some cst -> dep, pred, Globnames.destConstRef cst
```

Comment:
No destConstRef needed here, propagate the globref below instead.

Reply:
this code seems to be trying to refold schemes so it only works on constants, we modified the code to avoid the exn though